src/common/styles/cd-primary-button.less

```diff
@@ -0,0 +1,8 @@
+.primary-button() {
+  margin-top: 48px;
+  width: 200px;
```

Comment:
Becomes a problem when the text is long. I'd propose a min-width (with padding to ensure text doesn't go right to the edge)

Reply:
I realise now that this is copy/paste from elsewhere, but... 😅

Reply:
Fair still